pom.xml

```diff
@@ -138,12 +138,6 @@
         </exclusion>
       </exclusions>
     </dependency>
-    <dependency>
-    	<!-- Remove this dependency and the exclusion above when jenkins provides a up-to-date guava version (currently 11.0.1) -->
```

Comment:
[the exclusion](https://github.com/jenkinsci/jira-plugin/pull/404/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R321-R322) was not removed as per the comment.  if we remove it does that cause guava to be bundled incorrectly again?  (if so probably worth a comment here)

Reply:
no bundled guava 

```
➜  jira-plugin git:(guava-upgrade) ✗ unzip -l target/jira.hpi             
Archive:  target/jira.hpi
  Length      Date    Time    Name
---------  ---------- -----   ----
        0  01-23-2022 22:02   META-INF/
     1079  01-23-2022 22:02   META-INF/MANIFEST.MF
        0  01-23-2022 22:02   WEB-INF/
        0  01-23-2022 22:02   WEB-INF/lib/
      158  06-22-2021 12:49   help.html
      125  06-22-2021 12:49   help_de.html
      155  06-22-2021 12:49   help_ja.html
      348  06-22-2021 12:49   help-jira-create-issue.html
       87  06-22-2021 12:49   help-release-migrate.html
     9562  01-23-2022 22:02   WEB-INF/licenses.xml
   890168  06-23-2021 09:33   WEB-INF/lib/jaxb-impl-2.2.3-1.jar
   298176  01-23-2022 22:02   WEB-INF/lib/jira.jar
    23346  06-22-2021 21:48   WEB-INF/lib/stax-api-1.0-2.jar
    41513  07-30-2021 16:59   WEB-INF/lib/slf4j-api-1.7.32.jar
   634048  06-23-2021 12:43   WEB-INF/lib/joda-time-2.9.9.jar
   327135  07-21-2021 21:18   WEB-INF/lib/commons-io-2.11.0.jar
   105134  06-23-2021 09:33   WEB-INF/lib/jaxb-api-2.2.2.jar
    50042  06-23-2021 09:33   WEB-INF/lib/atlassian-event-3.0.0.jar
   263253  06-22-2021 21:42   WEB-INF/lib/plexus-utils-3.3.0.jar
    72903  06-23-2021 09:33   WEB-INF/lib/sal-api-3.0.7.jar
   134021  06-23-2021 09:33   WEB-INF/lib/jersey-client-1.19.jar
   139910  06-23-2021 09:33   WEB-INF/lib/atlassian-util-concurrent-4.0.1.jar
   202850  06-24-2021 07:57   WEB-INF/lib/jira-rest-java-client-core-5.2.2.jar
    46367  06-23-2021 09:33   WEB-INF/lib/jsr311-api-1.1.1.jar
    51644  06-23-2021 09:33   WEB-INF/lib/atlassian-httpclient-api-2.0.0.jar
   165345  06-23-2021 09:33   WEB-INF/lib/jersey-json-1.19.jar
    99282  06-23-2021 09:33   WEB-INF/lib/fugue-3.0.0.jar
    58076  11-24-2021 13:32   WEB-INF/lib/maven-artifact-3.8.4.jar
    67758  06-23-2021 09:33   WEB-INF/lib/jettison-1.1.jar
   436689  06-23-2021 09:33   WEB-INF/lib/jersey-core-1.19.jar
   587402  06-23-2021 07:06   WEB-INF/lib/commons-lang3-3.12.0.jar
   164932  06-24-2021 07:57   WEB-INF/lib/jira-rest-java-client-api-5.2.2.jar
    33031  06-23-2021 18:09   WEB-INF/lib/jsr305-3.0.0.jar
      165  06-22-2021 12:49   help_fr.html
       76  06-22-2021 12:49   help-release.html
       78  06-22-2021 12:49   help-version-create.html
        0  01-23-2022 22:02   META-INF/maven/
        0  01-23-2022 22:02   META-INF/maven/org.jenkins-ci.plugins/
        0  01-23-2022 22:02   META-INF/maven/org.jenkins-ci.plugins/jira/
     9660  01-23-2022 22:02   META-INF/maven/org.jenkins-ci.plugins/jira/pom.xml
      119  01-23-2022 22:02   META-INF/maven/org.jenkins-ci.plugins/jira/pom.properties
```

not sure this comment regarding the exclusion was not correct.